src/assets/styles/base.scss

```diff
@@ -264,7 +253,7 @@ ul.list {
 }
 
 .genErr {
-  font-size: 0.9rem;
+  font-size: 0.9rem; // 14.4px
```

Comment:
Even more unique styling! 
I imagine design calls for these.. but i am curious why so specific

Reply:
I didn't choose this font; I only commented it.

It's used in the dialogs. I've asked Tracey if we can change this to 14px or 15px.

Reply:
Tracey is OK with 15px. I have made the change.

Reply:
(14 looked too small and 16 looked too large. 15 is the Goldilocks number.)

Reply:
Perfect :)